fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/Payload/A320_251N/A320Payload.tsx

```diff
@@ -452,6 +462,28 @@ export const A320Payload: React.FC<A320Props> = ({
         gsxPayloadSyncEnabled,
     ]);
 
+    //If totalPax is 0, deactivate all sounds from the cabin
+    useEffect(() => {
+        let cabinSoundStatus:number = 0;
+        if(totalPax > 0)
+        {
+            cabinSoundStatus = 1;
+        }
+        if(globalSettingsRedux.passengerAmbienceActive)
+        {
+            setPassengerAmbienceEnabled(cabinSoundStatus);
+        }
+        if(globalSettingsRedux.cabinAnnouncementsActive)
+        {
+            setAnnouncementsEnabled(cabinSoundStatus);
+        }
+        if(globalSettingsRedux.boardindgMusicActive)
+        {
+            setBoardingMusicEnabled(cabinSoundStatus);
+        }
+        
+    },[totalPax]);
```

Comment:
```suggestion
    useEffect(() => {
        const cabinSoundStatus:number = (totalPax > 0 ? 1 : 0);

        if (globalSettingsRedux.passengerAmbienceActive) {
            setPassengerAmbienceEnabled(cabinSoundStatus);
        }

        if (globalSettingsRedux.cabinAnnouncementsActive) {
            setAnnouncementsEnabled(cabinSoundStatus);
        }

        if (globalSettingsRedux.boardindgMusicActive) {
            setBoardingMusicEnabled(cabinSoundStatus);
        }
    },[totalPax]);
```

Not realy sure of the conditions flow here.